### ISSUE TITLE
Update to latest blacklight and arclight

### ISSUE THREAD

**Issue by eliotjordan** [pulibrary/pulfalight#304]
- Blacklight v7.10.0
  - Currently on 7.2.0
- Arclight: v0.3.2
  - We're currently pinned to commit 6020c

As part of this upgrade, delete blacklight icon exclusions for search results page in `spec/features/accessibility_spec.rb`; problems here should be fixed in https://github.com/projectblacklight/blacklight/commit/575a0b40e921b75f01da5b9df84b4966a34c4b8d, 7.5.1 release

**Comment by hackartisan** [comment]
looks like this got done at some point.
